docs/content/content/types.md

```diff
@@ -60,6 +60,9 @@ Create a directory with the name of the type in `/layouts`. Type is always singu
 ### Create single template
 Create a file called `single.html` inside your directory. *E.g. `/layouts/post/single.html`*.
 
+### Create list template
+Create a file called `post.html` inside your directory. *E.g. `/layouts/section/post.html`*.
```

Comment:
I mean, I don't like the `inside your directory` part.  In the surrounding context, "your directory" is `/layouts/SECTION`.  It should say something like `inside the section lists template directory, "/layouts/section"`.